src/controllers/simplified-account/settings/switch-psp/make-test-payment/make-test-payment.controller.js

```diff
@@ -48,54 +60,48 @@ async function post (req, res, next) {
     })
 }
 
-async function getInbound (req, res, next) {
+async function getInbound(req, res, next) {
   const account = req.account
   const service = req.service
-  canStartTask(account, service)
   const user = req.user
   const chargeExternalId = req.session[VERIFY_PSP_INTEGRATION_CHARGE_EXTERNAL_ID_KEY]
   delete req.session[VERIFY_PSP_INTEGRATION_CHARGE_EXTERNAL_ID_KEY]
   const targetCredential = account.getSwitchingCredential()
   if (!chargeExternalId) {
     throw new Error('No charge found on session')
   }
-  chargeService.getCharge(service.externalId, account.type, chargeExternalId)
+  chargeService
+    .getCharge(service.externalId, account.type, chargeExternalId)
     .then(async (charge) => {
       if (charge.state.status === 'success') {
-        await worldpayDetailsService.updateCredentialState(service.externalId, account.type, targetCredential.externalId, user.externalId, CREDENTIAL_STATE.VERIFIED)
+        // TODO move me to more generic service
+        await worldpayDetailsService.updateCredentialState(
```

Comment:
Surely this is only if it's a worldpay service?

Reply:
no, we still need to set the credential to verified regardless of whether it is a stripe or worldpay credential